includes/service-providers/mailchimp/class-newspack-newsletters-mailchimp.php

```diff
@@ -704,7 +704,13 @@ public function get_send_lists( $args = [] ) {
 			}
 		}
 
-		return $send_lists;
+		// Convert to arrays.
+		return array_map(
+			function ( $list ) {
+				return $list->to_array();
+			},
+			$send_lists
+		);
```

Comment:
7a48870 refactors things so that the `get_send_lists()` methods now take a second optional arg, `$to_array`. If passed as `true`, the `Send_List` objects will be converted to arrays before returning, otherwise they'll be returned as regular objects. This lets us get the arrays for REST API and `localize_script` contexts and regular `Send_List` objects for PHP-only contexts.